puree/src/main/java/com/cookpad/puree/storage/PureeSQLiteStorage.java

```diff
@@ -110,6 +122,28 @@ public void delete(Records records) {
     }
 
     @Override
+    public void truncateBufferedLogs(int maxRecords) {
+        int recordSize = getRecordCount();
```

Comment:
Can you put `if (maxRecords == Integer.MAX_VALUE) return;` firstly? `Integer.MAX_VALUE` is the our use case.